### DISCUSSION TITLE
Doesn't open

### DISCUSSION THREAD

**Discussion by ghost** [koreader/koreader#11891] [Q&A]
PocketBook 633 Color 

I tried to install the nightly build from the application itself by Wi-Fi, but it went wrong (not as usual) and froze after I tapped install update button, so I closed the application. Then I tried to enter the KOReader again, the application did not open at all. Therefore, I downloaded this version from the site and unpacked it along the cord to the device, clicking replace identical files (I hope right?). But this didn’t help either; the application still doesn’t open at all. What to do? I can't delete and download again because I have important settings and files like patches there.

Update: I reinstalled and it works now. Closed.

**Comment by Frenzie** [comment]
Strange. I don't know how it works on PB. Maybe it was previously still stuck running or something?

**Reply by ghost** [reply]
> Strange. I don't know how it works on PB. Maybe it was previously still stuck running or something?

I don't know, before deleting I tried everything, like restarting, but it doesn't work anyway. So I saved different settings files, statistics folder and etc. and deleted KOReader, then I installed a stable version, put the saved in folders and started KOReader, it worked this time. Then I installed the new nightly build. Thanks for the nightly builds, I adore them!
